src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -96,6 +96,15 @@ class WorkspaceMembersPage extends React.Component {
             this.validate();
         }
 
+        if (!_.isEqual(prevProps.policyMemberList, this.props.policyMemberList)) {
+            this.setState((prevState) => {
+                const selectedEmployees = _.filter(prevState.selectedEmployees, (selectedEmployee) => _.has(this.props.policyMemberList, selectedEmployee));
+                return {
+                    selectedEmployees,
+                };
+            });
```

Comment:
Didn't you propose to use intersection? Is there something wrong with that?

```js
    this.setState((prevState) => ({
        selectedEmployees: _.intersection(prevState.selectedEmployees, this.props.policyMemberList),
    }));
```

Reply:
Right, this should do it then


```js
    this.setState((prevState) => ({
        selectedEmployees: _.intersection(prevState.selectedEmployees, _.keys(this.props.policyMemberList)),
    }));
```

Reply:
The main purpose of my solution is the way to fix this issue and this code above is for illustration purposes only.

Reply:
Can you apply the suggested fix